docs/src/api.md

```diff
@@ -0,0 +1,50 @@
+# API reference
```

Comment:
The docs structure does not have to follow the file structure for src.
In particular, overrides.jl as a docs section does not seem needed

Reply:
OK, I simplified it